### PR TITLE
Create discord subpage redirect

### PR DESCRIPTION
**Pull request by Darylgolden**
As per consensus in the Discord, I've created a subpage that redirects to the Discord invite link.

### CODE REVIEW

**Review by WampyCakes — APPROVED** [review]
Assuming that meta tag works correctly, LGTM

Is the empty body tag good practice or just unnecessary?

**Comment by WampyCakes** [comment]
The plugin site will also need the discord link updated

**Comment by Darylgolden** [comment]
Not sure, it's my first time making a HTML file for a project 😅

**Comment by WampyCakes** [comment]
My inclination is to remove it. I can't find anything online that says you _have_ to have it or that you _have_ to remove it 🤷‍♂️

**Comment by Darylgolden** [comment]
I've removed it.

File: discord/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="refresh" content="0; url='https://discord.gg/mMRrZQW'" />
+  </head>
+</html>
```
Comment (naveen521kk) [inline discord/index.html:6]:
```suggestion
</html>

```
Add a new line.

**Review by naveen521kk — APPROVED** [review]
sgtm

**Comment by naveen521kk** [comment]
let's see how it works in production.

**Comment by naveen521kk** [comment]
Works for me :).

**Comment by WampyCakes** [comment]
Same for me. Much easier way to tell people how to join the discord!

**Pull request merged**